brave/src/test/java/brave/features/baggage/BaggageInSingleHeaderTest.java

```diff
@@ -38,10 +38,7 @@
       .injectFormat(B3Propagation.Format.SINGLE).build())
       .add(SingleBaggageField.remote(field1))
       .add(SingleBaggageField.local(field2))
-      .add(InternalBaggage.instance.newBaggagePropagationConfig(
-          // BaggageCodec can see every baggage field, so it must be told what to not serialize
-          SingleHeaderCodec.newBuilder().blacklistField(field2).build()
-      ))
+      .add(InternalBaggage.instance.newBaggagePropagationConfig(SingleHeaderCodec.get()))
```

Comment:
this is the best part of the change. Now the codec needn't do work BaggagePropagation should be doing. Notably, it shouldn't ever even see local fields.